spec/e2e/id_property_spec.rb

```diff
@@ -73,7 +73,25 @@
       it 'will set the id_property after a session has been created' do
         node = Clazz.new
         expect(node).to respond_to(:the_id)
-        expect(Clazz.mapped_label.indexes[:property_keys].sort).to eq([[:the_id], [:uuid]])
+        expect(Clazz.mapped_label.indexes[:property_keys].sort).to eq([[:the_id]])
+      end
+    end
+
+    describe 'when having neo_id configuration' do
+      before do
+        before_session do
+          Neo4j::Config[:id_property] = :neo_id
+          stub_active_node_class('NeoIdTest')
+        end
+      end
+
+      it 'it will find node by neo_id' do
+        node = NeoIdTest.new
+        expect(node).to respond_to(:id)
+        expect(node).to respond_to(:neo_id)
+        node.save
+        expect(node.id).to eq(node.neo_id)
+        expect(NeoIdTest.where(id: node).first).to eq(node)
       end
```

Comment:
Can we remove some of these `expect` calls or split them into different examples?